frontend/models/mock-search-meta.tsx

```diff
@@ -0,0 +1,63 @@
+import { faCaretRight, faAngleRight, IconDefinition } from "@fortawesome/free-solid-svg-icons"
+
+export const resultsColumns = [
+  {
+    Header: "Search Date",
+    accessor: "searchDate" as const
```

Comment:
Would love to reuse table components - my saved search/results are made from copy-pasting the code from DataTable and then changing couple things. Didn't want to get my PR too far out of scope by fiddling with DataTable, but hoping somebody who's more comfortable with react-table would make something more generalized. 
Completely forgot about PrimaryInput, though!

Reply:
Oh hey, let's merge #152 first - there's a bunch of stuff I can use in there, plus it seems dumb to muck around with fake users when real ones (or at least fake users via API instead of mocks) are so close